### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by EryiXie** [dwofk/fast-depth#37]
Thanks for this great work. I am currently trying to train fast-depth with my own dataset. I have noticed there is not training scripts. So I would like to ask, which depth losses are used in training?

It would be very nice, If anyone can give me a suggestion about which losses should I pick.

**Comment by JVGD** [comment]
I was reading the paper and just wondering the same, came here and didn't find it either

**Comment by EryiXie** [comment]
> I was reading the paper and just wondering the same, came here and didn't find it either

Hi, I will begin to try some loss function design next week. Once I have some useful results, I will report it here.

**Comment by YiLiM1** [comment]
I read the paper and found that the author in the experimental part mentioned to follow the training method of "Sparse-to-dense: depth prediction from sparse depth samples and a single image"and L1 loss was used in that paper. I tried to train the network with L1 loss, and the result was very bad.

**Comment by JVGD** [comment]
I used the "depth loss" in [this paper](https://arxiv.org/pdf/1803.08673.pdf) and seems that the training is starting to converge.
![image](https://user-images.githubusercontent.com/6736158/99063114-4eb0b580-25a4-11eb-8965-937de6574631.png).

I also suspect that I have a shitty dataset, and that is why I am getting so many noise (although you can see the depth more or less in some sense in the high level, there is a lot of noise,  piwelwise)

**Comment by sunmengnan** [comment]
> I used the "depth loss" in [this paper](https://arxiv.org/pdf/1803.08673.pdf) and seems that the training is starting to converge.
> ![image](https://user-images.githubusercontent.com/6736158/99063114-4eb0b580-25a4-11eb-8965-937de6574631.png).
> 
> I also suspect that I have a shitty dataset, and that is why I am getting so many noise (although you can see the depth more or less in some sense in the high level, there is a lot of noise, piwelwise)

which dataset are you using?
